CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.66.0
+
+- When configuring Copr chroot (target in Packit terminology) specific configuration, make sure to specify `additional_modules` as a string: module names separated with a comma, example: "httpd:2.4,python:4". (#1826)
+- Target-specific configuration for Copr builds can now be defined and Packit will set it for the appropriate Copr chroots. (#1822)
+- You can now specify `update_release: false` in the configuration to tell Packit not to change the `Version` and `Release` in the spec file. It works the same as `--no-update-release` (renamed from now deprecated `--no-bump`) in the CLI. (#1827)
+- Packit now supports setting `module_hotfixes` for Copr projects. (#1829)
+- All Copr projects created by Packit now default to `enable_net=False`: our documentation stated this but it wasn't the case. This is now corrected. (#1825)
```

Comment:
Again, the colon, am I the only one having a problem with it?
```suggestion
- All Copr projects created by Packit now default to `enable_net=False`. Our documentation stated this but it wasn't the case. This is now corrected. (#1825)
```

Reply:
No, you are not the only one, I should have fixed that right away. Thanks for the suggestions.

Reply:
I did not parse it, what are you using for the validation?
I just saw the preview of the file given by Github...